packages/flutter/lib/src/material/colors.dart

```diff
@@ -73,10 +94,23 @@ class MaterialColor extends ColorSwatch {
 ///
 ///  * [Colors], which defines all of the standard material colors.
 class MaterialAccentColor extends ColorSwatch {
+  /// Creates a color swatch with a variety of shades appropriate for accent
+  /// colors.
```

Comment:
I linked to the spec, but we might need some more canonical description in ThemeData somewhere to link to.